pupil_src/shared_modules/calibration_routines/calibrate.py

```diff
@@ -398,6 +392,48 @@ def fn(pt_0, pt_1):
     return fn
 
 
+def match_data(g_pool, pupil_list, ref_list):
+    if pupil_list and ref_list:
+        pass
+    else:
+        logger.error(not_enough_data_error_msg)
+        return {
+            "subject": "calibration.failed",
+            "reason": not_enough_data_error_msg,
+            "timestamp": g_pool.get_timestamp(),
+            "record": True,
+        }
+
+    # match eye data and check if biocular and or monocular
```

Comment:
```suggestion
    # match eye data and check if binocular and or monocular
```

Reply:
Does this comment make sense and/or is it needed? In what sense is eye data being matched? Isn't it rather gaze and pupil data being matched? 
What is checked for being monocular or binocular? We are always computing both versions anyway.

Reply:
Yeah, I agree. I have removed the comment.